Makefile

```diff
@@ -795,6 +790,18 @@ eve: $(INSTALLER) $(EVE_ARTIFACTS) current $(RUNME) $(BUILD_YML) | $(BUILD_DIR)
 	fi
 	$(QUIET): $@: Succeeded
 
+EVE_VERIFICATION_ARTIFACTS=$(BIOS_IMG) $(EFI_PART) $(CONFIG_IMG) $(PERSIST_IMG) $(INITRD_IMG) $(VERIFICATION_IMG) $(ROOTFS_IMG) $(SBOM) $(BSP_IMX_PART) fullname-rootfs $(BOOT_PART)
+eve-verification: $(VERIFICATION) $(EVE_VERIFICATION_ARTIFACTS) current | $(BUILD_DIR)
```

Comment:
Hmm, I guess the target is called eve-verification, but that seems counterintuitive. Isn't "verification" a better name?